wurst/file/ByteBuffer.wurst

```diff
@@ -0,0 +1,145 @@
+package ByteBuffer
+
+import Table
+import Bitwise
+
+/*	ByteBuffer is a utility class that allows you to write/read bytes, shorts and integer to/from it.
```

Comment:
What is the analog class in - say - Java?

What's the endianness?

Reply:
`ByteBuffer` is something similar in Java, though has more functionality.
Integers bigger than byte are treated as in BigEndian.

Reply:
I think the documentation needs that context, not the PR